pkg/vm/stack.go

```diff
@@ -127,7 +127,9 @@ func (e *Element) Bytes() []byte {
 		if t.value {
 			return []byte{1}
 		}
-		return []byte{0}
+		// return []byte{0}
+		// FIXME revert when NEO 3.0
```

Comment:
vote to create an issue and write issue number here so we can track that

Reply:
done